freqtrade/strategies/CprCamarillaSmaStrategy.py

```
@@ -0,0 +1,205 @@
+        dataframe.loc[:, ['exit_long', 'exit_short']] = (0, 0)
+        return dataframe
+
+    def custom_leverage(self, pair: str, current_time: 'datetime', current_rate: float,
+                        proposed_leverage: float, max_leverage: float, side: str,
+                        **kwargs) -> float:
+        """
+        Sets a custom leverage for each trade.
+        """
+        return 3.0
+
+    def confirm_trade_entry(self, pair: str, order_type: str, amount: float, rate: float,
```

Ruff (0.13.1)

Unused method argument: `order_type`

(ARG002)